### ISSUE TITLE
unfixed startvalues in Modelica.Electrical.Analog.Examples.AD_DA_conversion

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#1563]
**Reported by jfrenkel on 2 Sep 2014 06:20 UTC**
In the example Modelica.Electrical.Analog.Examples.AD_DA_conversion are the following unfixed Variables in the initial System:
 * `pulse.T0`
 * `dA_Converter.vout`

The following Variables could also be set `fixed=true` but they does not infulence the initial system and could be ignored because of that:
 * `aD_Converter.z`
 * `dA_Converter.y`

OpenModelica reports: 
```
Warning: Assuming fixed start value for the following 4 variables:
         dA_Converter.vout:DISCRETE(fixed = false ) .Modelica.Electrical.Analog.Examples.AD_DA_conversion, .Modelica.Electrical.Analog.Ideal.DA_Converter, .Modelica.SIunits.Voltage type: Real 
         pulse.T0:DISCRETE(start = pulse.startTime fixed = false )  "Start time of current period".Modelica.Electrical.Analog.Examples.AD_DA_conversion, .Modelica.Electrical.Digital.Sources.Pulse, .Modelica.SIunits.Time type: Real 
         aD_Converter.z:DISCRETE(fixed = false ) .Modelica.Electrical.Analog.Examples.AD_DA_conversion, .Modelica.Electrical.Analog.Ideal.AD_Converter, .Integer type: Integer 
         dA_Converter.y:DISCRETE(start = 0.0 fixed = false ) .Modelica.Electrical.Analog.Examples.AD_DA_conversion, .Modelica.Electrical.Analog.Ideal.DA_Converter, .Real type: Real
```

and also SimX report it. Dymola does not report it. 

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/1563

**Comment by modelica-trac-importer** [comment]
**Modified by beutlich on 8 Sep 2014 11:27 UTC**

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 6 Jan 2015 16:37 UTC**
Fixed in 3fc5a9b549775b8a7afa59917c364f91fb85bcee (maintenance) and in 1c23a931f033b6f94e462e6552345f134d46e0bf (trunk):

In fact fixed=true definitions are missing. Therefore, the following changes have been performed:

- Modelica.Electrical.Digital.Sources.Pulse: Introduced fixed=true for T0
- Modelica.Electrical.Analog.Ideal.DA_Converter: Introduced for y "fixed = true" and for vout "start=0, fixed = true"
- Modelica.Electrical.Analog.Ideal.AD_Converter: Introduced for z "start=0, fixed=true"

**Comment by modelica-trac-importer** [comment]
**Changelog modified by otter on 6 Jan 2015 16:37 UTC**
Digital.Sources.Pulse, DA_Converter, AD_Converter: Adding missing fixed=true values for discrete-time variables
